home_work_2_zd_3.js

```diff
@@ -0,0 +1,15 @@
+let str = "JavaScript";
+
+function reverse(str) {
```

Comment:
function is not working properly. For example if we execute function with the string that contents numbers.

Reply:
Done